### PR TITLE
Fix azure compute options

### PR DESCRIPTION
**Pull request by luisgmsft**
Please review and comment.

### CODE REVIEW

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/guide/technology-choices/compute-comparison.md](https://github.com/luisgmsft/architecture-center/blob/fix-azure-compute-options/docs/guide/technology-choices/compute-comparison.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/guide/technology-choices/compute-comparison?branch=pr-en-us-304) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C11%5C21%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201711212036075147-304%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=LWUn0AU5cXXlPr0yomjbtDeImfAo%2FkMBYIinHggobrA%3D&st=2017-11-21T20%3A32%3A15Z&se=2017-12-22T20%3A37%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/guide/technology-choices/compute-comparison.md [hunk 1]
```diff
@@ -13,12 +13,12 @@ The term *compute* refers to the hosting model for the computing resources that
...
 |----------|-----------------|-------------|----------------|-----------------|-------------------------|----------------|-------------|
-| Application composition | Agnostic | Applications | Services, guest executables | Functions | Containers | Roles | Scheduled jobs  |
+| Application composition | Agnostic | Applications, Linux Containers | Services, guest executables, containers | Functions | Linux Containers, Windows Containers (preview) | Roles | Scheduled jobs  |
 | Density | Agnostic | Multiple apps per instance via app plans | Multiple services per VM | No dedicated instances <a href="#note1"><sup>1</sup></a> | Multiple containers per VM | One role instance per VM | Multiple apps per VM |
-| Minimum number of nodes | 1 <a href="#note2"><sup>2</sup></a>  | 1 | 5 <a href="#note3"><sup>3</sup></a> | No dedicated nodes <a href="#note1"><sup>1</sup></a> | 3 | 2 | 1 <a href="#note4"><sup>4</sup></a> |
+| Minimum number of nodes | 1 <a href="#note2"><sup>2</sup></a>  | 1 | 5 <a href="#note3"><sup>3</sup></a>. Azure VM Scale Set supported | No dedicated nodes <a href="#note1"><sup>1</sup></a> | 3. Azure VM Scale Set not supported | 2 | 1 <a href="#note4"><sup>4</sup></a> |
```
Comment (MikeWasson) [inline docs/guide/technology-choices/compute-comparison.md]: Scale sets is covered under scalability section (auto-scaling)

File: docs/guide/technology-choices/compute-comparison.md [hunk 2]
```diff
@@ -13,12 +13,12 @@ The term *compute* refers to the hosting model for the computing resources that
...
 | State management | Stateless or Stateful | Stateless | Stateless or stateful | Stateless | Stateless or Stateful | Stateless | Stateless |
-| Web hosting | Agnostic | Built in | Self-host, IIS in containers | Not applicable | Agnostic | Built-in (IIS) | No |
-| OS | Windows, Linux | Windows, Linux (preview)  | Windows, Linux  (preview) | Not applicable | Windows,  Linux | Windows | Windows, Linux |
+| Web hosting | Agnostic | Built in | Self-host, IIS | Not applicable | Self-host, IIS | Built-in (IIS) | No |
```
Comment (MikeWasson) [inline docs/guide/technology-choices/compute-comparison.md]: For ACS, I think we should leave this as "agnostic" - you can run anything (HAProxy, Nginx, Tomcat)

File: docs/guide/technology-choices/compute-comparison.md [hunk 3]
```diff
@@ -13,12 +13,12 @@ The term *compute* refers to the hosting model for the computing resources that
...
+| OS | Windows, Linux | Windows, Linux  | Windows, Linux | Not applicable | Windows,  Linux | Windows | Windows, Linux |
```
Comment (MikeWasson) [inline docs/guide/technology-choices/compute-comparison.md]: For ACS add "(preview)" to Windows

File: docs/guide/technology-choices/compute-comparison.md (hunk 1 above)
Comment (luisgmsft) [reply docs/guide/technology-choices/compute-comparison.md]: Done.

File: docs/guide/technology-choices/compute-comparison.md (hunk 2 above)
Comment (luisgmsft) [reply docs/guide/technology-choices/compute-comparison.md]: Done.

File: docs/guide/technology-choices/compute-comparison.md (hunk 3 above)
Comment (luisgmsft) [reply docs/guide/technology-choices/compute-comparison.md]: Done.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/guide/technology-choices/compute-comparison.md](https://github.com/luisgmsft/architecture-center/blob/fix-azure-compute-options/docs/guide/technology-choices/compute-comparison.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/guide/technology-choices/compute-comparison?branch=pr-en-us-304) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C11%5C21%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201711212119078529-304%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=dQ5UqYm5JwVidRfSWjGtsFZXnVPtgKQrOV6HFsqgNFg%3D&st=2017-11-21T21%3A15%3A01Z&se=2017-12-22T21%3A20%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/guide/technology-choices/compute-comparison.md](https://github.com/luisgmsft/architecture-center/blob/fix-azure-compute-options/docs/guide/technology-choices/compute-comparison.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/guide/technology-choices/compute-comparison?branch=pr-en-us-304) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C11%5C22%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201711221812218890-304%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=sX5SqLq1GpTYSPxcvadJDfTSVo19%2FkXgfiEIbCcACIY%3D&st=2017-11-22T18%3A08%3A26Z&se=2017-12-23T18%3A13%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**